.gitmodules

```diff
@@ -20,4 +20,4 @@
 [submodule "windows_docker_resources/ros2-cookbooks"]
 	path = windows_docker_resources/ros2-cookbooks
 	url = git@github.com:ros-infrastructure/ros2-cookbooks
-	branch = latest
+	branch = clalancette/remove-tinyxml
```

Comment:
Note that before merging, we need to switch this back to "latest" and update the hash.